dask/dataframe/io/parquet/arrow.py

```diff
@@ -1042,7 +1042,7 @@ def _collect_dataset_info(
         }
 
     @classmethod
-    def _create_dd_meta(cls, dataset_info, use_nullable_dtypes=False):
+    def _create_dd_meta(cls, dataset_info, use_nullable_dtypes=False, **kwargs):
```

Comment:
Good point, I tried with `geopandas`, and yes it will need a corresponding change. I'll submit a PR.

Reply:
Easiest solution is just to add what you need to the `dataset_info` dictionary in `_collect_dataset_info`. In fact, that is probably what we *should* have done for `use_nullable_dtypes`. Note that `dataset_info` is just a rough way to avoid plumbing a large number of optional parameters through all the static/class methods here. I could be mistaken, but my current intuition is to avoid using `**kwargs` at all costs.

Reply:
Thanks @rjzamora, I reverted the change to `create_dd_meta`. Now it doesn't break with `dask-geopandas`.